src/gui/integration/integration_test.go

```diff
@@ -215,7 +235,11 @@ func TestStableOutputs(t *testing.T) {
 			require.NoError(t, err)
 
 			var expected visor.ReadableOutputSet
-			loadJSON(t, tc.golden, &expected)
+			goldenFile := filepath.Join("test-fixtures", tc.golden)
```

Comment:
This can't be done automatically in `loadJSON`?

Reply:
using the filename in `updateGoldenFile` as well